Resources/Prototypes/_EE/Entities/Clothing/OuterClothing/jackets.yml

```
@@ -3,13 +3,28 @@
+- type: entity
+  parent: ClothingOuterBase
+  id: ClothingOuterJacketSurgical
+  name: surgical jacket
+  description: A thick jacket with a membrane attached to the end of its sleeves that conforms to the user's hands, allowing it to function as surgical gloves. Useful for if you need to do surgery but can't wear normal gloves.
+  components:
+  - type: Sprite
+    sprite: _EE/Clothing/GloveJackets/surgical_jacket.rsi
+  - type: Clothing
+    sprite: _EE/Clothing/GloveJackets/surgical_jacket.rsi
+  - type: FingerprintMask
+  - type: GloveJacket
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)